src/_includes/components/card.macro.html

```diff
@@ -43,7 +43,8 @@ <h3 class="card__title">
             {{ params.teaser }}
         </p>
         <div class="card__footer">
-            <img class="blog-post__author-photo" src="{% if params.authorImage %}{{ params.authorImage }} {% else %}../../../assets/images/people/_placeholder-avatar.svg{% endif %}" alt="{{ params.authorName }}" width="40" height="40" loading="lazy">
+            {% set fallback_image = '/assets/images/people/_placeholder-avatar.svg' %}
+            <img class="blog-post__author-photo" src="{{ params.image if params.image else fallback_image }}" alt="{{ params.authorName }}" width="40" height="40" loading="lazy" onerror="this.src = '{{fallback_image}}'">
```

Comment:
>I'm not sure we need the fallback changes.

We can see here a fallback image is already applied but actually it's not working, can it be considered as a bug?

Reply:
Yes, I think that can be considered a bug. I just don't think we need to add fallback images to places where we didn't have them before.

Reply:
I guess the fallback image was working as intended.

Reply:
Fallback images was only working for sponsors in `All backers` section as you can see in images provided in issue template of https://github.com/eslint/eslint.org/issues/679

Reply:
>I just don't think we need to add fallback images to places where we didn't have them before.

reverted the changes.